recruitments/models.py

```diff
@@ -0,0 +1,58 @@
+from ctypes import addressof
+from pyexpat import model
+from sqlite3 import Timestamp
+from django.db   import models
+from core.models import TimeStampModel
+
+class OccupationCategory(TimeStampModel):
+    name = models.CharField(max_length=50)
+    
+    class Meta:
+        db_table = 'occupation_categories'
+
+class OccupationSubcategory(TimeStampModel):
+    name                = models.CharField(max_length=50)
+    occupation_category = models.ForeignKey('OccupationCategory', on_delete=models.CASCADE, related_name='occupation_subcategories')
+    
+    class Meta:
+        db_table = 'occupation_subcategories'
+
+class Recruitment(TimeStampModel):
+    name                   = models.CharField(max_length=50)
+    description            = models.TextField(max_length=2000)
+    occupation_subcategory = models.ForeignKey('OccupationSubcategory', on_delete=models.CASCADE)
+    company                = models.ForeignKey('companies.Company', on_delete=models.CASCADE, related_name='recruitments')
+    deadline               = models.DateField()
+    address                = models.CharField(max_length=200)           
+
+    class Meta:
+        db_table = 'recruitments'
+
+class UserRecruitment(TimeStampModel):
+    user               = models.ForeignKey('OccupationSubcategory', on_delete=models.CASCADE, related_name='user_recruitments')
+    recruitment        = models.ForeignKey('Recruitment', on_delete=models.CASCADE, related_name='user_recruitments')
+    apllication_status = models.ForeignKey('ApllicationStatus', on_delete=models.CASCADE, related_name='user_recruitments')
+    
+    class Meta:
+        db_table = 'user_recruitments'
+
+class ResumeUserRecruitment():
+    user_recruitment = models.ForeignKey('UserRecruitment', on_delete=models.CASCADE, related_name='resume_user_recruitments')
+    resume           = models.ForeignKey('resumes.Resume', on_delete=models.CASCADE, related_name='resume_user_recruitments')
+    
+    class Meta:
+        db_table = 'resume_user_recruitments'
```

Comment:
- () 상속을 받지 않으시겠다...???!!
```suggestion
class ResumeUserRecruitment():
    user_recruitment = models.ForeignKey('UserRecruitment', on_delete=models.CASCADE, related_name='resume_user_recruitments')
    resume           = models.ForeignKey('resumes.Resume', on_delete=models.CASCADE, related_name='resume_user_recruitments')
    
    class Meta:
        db_table = 'resume_user_recruitments'
```

Reply:
수정했습니다